.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: Rust
+
+on: [push]
+
+jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: Osspial/setup-rust-action@windows-uninstall-default-stable
+      with:
+        rust-version: stable
+        components: rustfmt
+    - name: Check Formatting
+      run: cargo +stable fmt --all -- --check
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        rust_version: [stable]
```

Comment:
Test nightly too?

Reply:
I'll test the other channels once this is ready to merge. I'd cut it back to just stable to speed up the edit/debug cycle.